### PR TITLE
dropbear: close all active client connections on shotdown

### PR DESCRIPTION
**Pull request by chris5560**
On system shutdown client connections to dropbear stay open.

Signed-off-by: Christian Schoenebeck <<christian.schoenebeck@gmail.com>>

### CODE REVIEW

**Comment by alive4ever** [comment]
> On system shutdown client connections to dropbear stay open.

Issuing `poweroff; exit` closes the ssh connection before shutdown.

File: package/network/services/dropbear/files/dropbear.init [hunk 1]
```diff
@@ -148,6 +148,11 @@ service_triggers()
 	procd_add_validation validate_section_dropbear
 }
 
+shutdown() {
+	# close all open connections
+	killall dropbear
```
Comment (alive4ever) [inline package/network/services/dropbear/files/dropbear.init]: `killall` sounds dangerous.

How about `pkill dropbear` or `kill -TERM $(pgrep dropbear)`?

Or even better, how about a test to determine whether dropbear process is still running?
`pgrep dropbear && kill -TERM $(pgrep dropbear)`

**Comment by wongsyrone** [comment]
Anyone check if the issue is fixed by new release? https://github.com/openwrt/openwrt/pull/789

**Comment by rockdrilla** [comment]
Hi @wongsyrone!
Yes, I've checked new release against this feature (see PR 876).
No, this issue isn't fixed by new version (is there issue/PR in upstream?).

**Comment by rotanid** [comment]
why was this closed? :(

**Comment by chris5560** [comment]
I never closed this.
Whatever happend here.
I'm also not able to reopen this pull.
I'm try this directly via GitHub webpage

**Comment by ysc3839** [comment]
I found the lost commit: https://github.com/openwrt/openwrt/commit/1ebfac583a292074879bf460197ca138fb8f926e

**Pull request closed without merging**